src/main/java/gregtech/common/metatileentities/storage/MetaTileEntityQuantumTank.java

```diff
@@ -323,6 +326,22 @@ protected ModularUI createUI(EntityPlayer entityPlayer) {
                 .build(getHolder(), entityPlayer);
     }
 
+    private Consumer<List<ITextComponent>> getFluidNameText(TankWidget tankWidget) {
+        return (list) -> {
+            String fluidName = tankWidget.getFluidLocalizedName().isEmpty() ? this.lockedFluid != null ? this.lockedFluid.getLocalizedName() : "" : tankWidget.getFluidLocalizedName();
```

Comment:
In the cases where this ternary and the one below result in `""`, should adding to the list be prevented entirely, instead of adding the empty string?